vagrant-centos-kubeadm-v1.9.2/Dockerfile.pause

```diff
@@ -0,0 +1,3 @@
+FROM microsoft/windowsservercore-insider
```

Comment:
Is this only intended for use with insider builds? So far this repo is targeting 1709 builds and would need to call out if that is an expected requirement

Reply:
Yes, we have several bug fixes and updates which are going to target WS 1803. I'll update to reference the public, insider builds